src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
All magic numbers in your code should be constants. Here, the number of possible colors is not explicitly defined, which can lead to confusion. Please create a constant field to hold this value. You should declare it like this: `private static final int COLORS_COUNT = Color.values().length;` and then use it in your code: `new Random().nextInt(COLORS_COUNT);`